### PR TITLE
only copy apiserver certs to control plane

### PR DESCRIPTION
**Pull request by sharifelgamal**

### CODE REVIEW

File: pkg/minikube/bootstrapper/certs.go [hunk 1]
```diff
@@ -56,9 +56,12 @@ func SetupCerts(cmd command.Runner, k8s config.KubernetesConfig, n config.Node)
 		return nil, errors.Wrap(err, "shared CA certs")
 	}
 
-	xfer, err := generateProfileCerts(k8s, n, ccs)
-	if err != nil {
-		return nil, errors.Wrap(err, "profile certs")
+	var xfer []string
```
Comment (medyagh) [inline pkg/minikube/bootstrapper/certs.go:59]: what is xfer short for ?

Comment (sharifelgamal) [reply pkg/minikube/bootstrapper/certs.go:59]: that's the name it already had, believe it's short of transfer?

**Comment by sharifelgamal** [comment]
The test failures are all accounted for in existing issues. KVM is old k8s version, vbox linux is TestErrorSpam.

**Pull request merged**